tests/installation/confirm_installation.pm

```diff
@@ -9,9 +9,21 @@
 
 use strict;
 use warnings;
+use testapi;
 use base 'y2_installbase';
+use version_utils 'is_sle';
 
 sub run {
+    # For SLE 15 SP4 tests that have the Legacy module added during instalaltion, there is
+    # additional licence popup that is handled by the following function.
+    if ((get_var("SCC_ADDONS") =~ /legacy/) && (is_sle("=15-SP4"))) {
```

Comment:
we need a ticket to remove this condition:
the product part of the condition via our distribution pattern and the first part just passing as parameter to a future confirm_installation_method() as we discussed in Slack.

Reply:
I will open a ticket for this, but can't say that I have full understanding of your solution proposal. You want this java popup to be handled inside the confirm_installation() which will receive some arguments and act accordingly? The conditions should be inside the library?

Reply:
Can you please, check here: https://progress.opensuse.org/issues/127529

Reply:
I rephrased a bit but you got the idea, thanks!